ui/packages/catalog-ui-search/src/main/webapp/react-component/container/marionette-region-container/marionette-region-container.tsx

```diff
@@ -41,10 +42,13 @@ export default hot(module)(
         this.region.show(this.props.view, {
           replaceElement: this.props.replaceElement,
         })
+        this.props.bindView && this.props.bindView(this.props.view)
```

Comment:
These changes are the same as here: https://github.com/codice/ddf/pull/4476 whichever is merged first, the other PR will require a rebase